### PR TITLE
qTox now uses default smiley pack if there's an invalid one in the qtox.ini file (fixes #2757)

### LINKED ISSUE #2757: qTox should use default smiley pack if there's an invalid one in the qtox.ini file
Hello,

if there's an invalid/unavailable smiley pack in the qtox.ini file, qTox should automatically choose the default one.

So, the user don't have to go to the settings and moreover this change makes the settings more reliable also for the future.

### PR DESCRIPTION
**Pull request by tgharib**
https://github.com/tux3/qTox/issues/2757 for quick reference

### CODE REVIEW

File: src/persistence/settings.cpp [hunk 1]
```diff
@@ -192,7 +192,12 @@ void Settings::loadGlobal()
     s.endGroup();
 
     s.beginGroup("GUI");
-        smileyPack = s.value("smileyPack", ":/smileys/Universe/emoticons.xml").toString();
+        static const QString DEFAULT_SMILEYS = ":/smileys/Universe/emoticons.xml";
```
Comment (antis81) [inline src/persistence/settings.cpp]: remove the "static" here.

**Comment by tgharib** [comment]
Ping. Static modifier has been removed.

**Comment by tux3** [comment]
Thanks, it works great.

**Comment by antis81** [comment]
:+1:

**Pull request merged**